modules/axios/plugin.js

```diff
@@ -118,6 +136,23 @@ export default (ctx) => {
   });
   <% } %>
 
+  <% if(options.debug) { %>
+  axios.interceptors.request.use(config => {
+    debug('[@nuxtjs/axios] Request:', config)
+    return config
+  }, error => {
+    debug('error', '[@nuxtjs/axios] Error:', error)
+    return Promise.reject(error)
+  });
+  axios.interceptors.response.use(config => {
+    debug('[@nuxtjs/axios] Response:', config)
+    return config
+  }, error => {
+    debug('error', '[@nuxtjs/axios] Error:', error)
+    return Promise.reject(error)
```

Comment:
Do you confirm this rejection is handled with plugin error handler? (next interceptor)